effect_prn/form_validators/end_of_study.py

```diff
@@ -2,64 +2,47 @@
     RequiresDeathReportFormValidatorMixin,
 )
 from edc_consent.constants import CONSENT_WITHDRAWAL
-from edc_constants.constants import DEAD, OTHER
+from edc_constants.constants import OTHER, YES
 from edc_form_validators import FormValidator
-from edc_ltfu.constants import LOST_TO_FOLLOWUP
-from edc_ltfu.modelform_mixins import RequiresLtfuFormValidatorMixin
 
-from ..constants import OTHER_RX_DISCONTINUATION
+from ..constants import CARE_TRANSFERRED_OUT, LATE_EXCLUSION_OTHER
 
 
 class EndOfStudyFormValidator(
     RequiresDeathReportFormValidatorMixin,
-    RequiresLtfuFormValidatorMixin,
     FormValidator,
 ):
     death_report_model = "effect_ae.deathreport"
     ltfu_model = None
 
     def clean(self):
 
-        self.validate_death_report_if_deceased()
-        self.validate_ltfu()
+        self.required_if(YES, field="cm_admitted", field_required="cm_admitted_cnt")
 
-        if self.cleaned_data.get("offschedule_reason"):
-            if self.cleaned_data.get("offschedule_reason").name != OTHER:
-                self.validate_other_specify(
-                    field="offschedule_reason",
-                    other_specify_field="other_offschedule_reason",
-                    other_stored_value=OTHER_RX_DISCONTINUATION,
-                )
-
-            if self.cleaned_data.get("offschedule_reason").name != OTHER_RX_DISCONTINUATION:
-                self.validate_other_specify(
-                    field="offschedule_reason",
-                    other_specify_field="other_offschedule_reason",
-                    other_stored_value=OTHER,
-                )
-
-        self.required_if(DEAD, field="offschedule_reason", field_required="death_date")
+        self.required_if(
+            OTHER, field="offschedule_reason", field_required="offschedule_reason_other"
+        )
 
         self.required_if(
-            LOST_TO_FOLLOWUP,
+            CONSENT_WITHDRAWAL,
             field="offschedule_reason",
-            field_required="ltfu_date",
+            field_required="withdrawal_consent_reasons",
         )
 
         self.required_if(
-            CONSENT_WITHDRAWAL,
+            LATE_EXCLUSION_OTHER,
             field="offschedule_reason",
-            field_required="consent_withdrawal_reason",
+            field_required="late_exclusion_other",
         )
 
         self.required_if(
-            "included_in_error",
+            CARE_TRANSFERRED_OUT,
             field="offschedule_reason",
-            field_required="included_in_error",
+            field_required="transferred_consent",
         )
 
         self.required_if(
-            "included_in_error",
+            CONSENT_WITHDRAWAL,
             field="offschedule_reason",
-            field_required="included_in_error_date",
+            field_required="consent_withdrawal_reason",
         )
```

Comment:
What's the difference between this, and lines 26-30, `withdrawal_consent_reasons`?